### PR TITLE
Minor text cleanup

### PR DESCRIPTION
**Pull request by dj2**
This CL cleans up a bit of spelling, some formatting and a couple missing copyright headers.

### CODE REVIEW

File: PRINCIPLES.md [hunk 1]
```diff
@@ -107,38 +106,38 @@ This library is an experiment and not intended for use. See the
...
       in smaller projects.
     * We will move implementations out of headers, and out of templates, to
       reduce header sizes where possible and where it does not have strong
       performance implications.
-1. We will use doxygen to document all types, functions, and public methods.
+1. We will document all types, functions, and public methods.
     * Documentation will include examples where use is non-trivial.
-    * All functions that can abort() will document under what conditions they do
-      so, with the exception of: use-after-move.
+    * All functions that can a`bort()` will document under what conditions they
```
Comment (danakj) [inline PRINCIPLES.md:126]: misplaced `

File: PRINCIPLES.md [hunk 2]
```diff
@@ -107,38 +106,38 @@ This library is an experiment and not intended for use. See the
...
+      do so, with the exception of: use-after-move.
 1. No null pointers. Smart pointers have no null state. Functions never return
-    null pointers.
+   null pointers.
 1. No raw pointers by default. Raw pointers throw away all lifetime information,
```
Comment (danakj) [inline PRINCIPLES.md:130]: i haven't lived up to this one, cuz i decided to try make raw pointers safe instead but yeah just fyi.

File: PRINCIPLES.md (hunk 1 above)
Comment (dj2) [reply PRINCIPLES.md]: Done, nice catch.

File: PRINCIPLES.md (hunk 2 above)
Comment (dj2) [reply PRINCIPLES.md:130]: Ack, that sounds like a separate issue from this PR, heh.

**Comment by danakj** [comment]
How did choice_unittest break compiling?

File: PRINCIPLES.md (hunk 2 above)
Comment (danakj) [reply PRINCIPLES.md:130]: yeah definitely :)

**Comment by dj2** [comment]
I haven't been able to figure that out. I was going to as, are the Nightly bots pulling from tip-of-tree? The latest bots are still building correctly, so is this an issue with latest LLVM, or this CL?

**Comment by danakj** [comment]
How it works is:
- The "latest bots" just build TOT when something lands.
- The bots on the PR are building the PR against TOT.

But yes the clang support for aggregate initialization was reverted today, that must be it, argh.

**Pull request merged**